include/url.h

```diff
@@ -63,6 +63,11 @@ namespace Url
 
         const std::string& userinfo() const { return userinfo_; }
 
+        /**
+         * Get a new string representation of the URL.
+         **/
+        std::string toString() const;
```

Comment:
In C++, this method is often just called [`str()`](http://en.cppreference.com/mwiki/index.php?title=Special%3ASearch&search=str).